cmake/Install.cmake

```diff
@@ -404,6 +408,8 @@ endif()
 #set(CPACK_IFW_PACKAGE_WIZARD_DEFAULT_HEIGHT 480)
 set(CPACK_IFW_PACKAGE_WINDOW_ICON "${CMAKE_SOURCE_DIR}/release/ep_nobg.png")
 
+# TODO: TEMP
+set(CPACK_IFW_VERBOSE ON)
```

Comment:
Is this `TEMP` supposed to be removed before merge?

Reply:
While these things are changing, I'm just going to leave this in.  I took off the TODO statement.

Reply:
I actually like seeing the verbose output when building package, but not sure everyone does.